.github/workflows/_common.yml

```diff
@@ -1,4 +1,4 @@
-name: (internal) - Build, Lint, Test
+name: (abstract) - Build, Lint, Test
```

Comment:
just for information: whats the difference between internal and abstract?

Reply:
Mostly just naming - was switched to `abstract` to be more accurate _(aka the workflow isn't meant to be triggered on its own, only called from other workflows)_